compiam/conf/raga/carnatic.yaml

```diff
@@ -1,35 +1,295 @@
+Melakarta Ragam 
```

Comment:
I would leave this as a comment... at the moment the yaml will not load as it is currently formated.

You can try by using the load_yaml function in compiam.utils

Reply:
So it will remain in the file like this...

```# Melakarta Ragam```

And then we can unindent the rest

Reply:
Actually I would create two separate files.... melakarta.yaml with this information and another for the children ragas (can you think of an appropriate name?)